src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -55,36 +55,25 @@ public sealed class SonarQubeIssuesProvider : ISonarQubeIssuesProvider
         private CancellationTokenSource cancellationTokenSource;
         private readonly IThreadHandling threadHandling;
 
-        public SonarQubeIssuesProvider(ISonarQubeService sonarQubeService, string sonarQubeProjectKey, ITimerFactory timerFactory,
-            ILogger logger) : this(sonarQubeService, sonarQubeProjectKey, timerFactory, logger, new ThreadHandling())
+        public SonarQubeIssuesProvider(ISonarQubeService sonarQubeService, string sonarQubeProjectKey,
+            ILogger logger) : this(sonarQubeService, sonarQubeProjectKey, logger, new TimerFactory(), new ThreadHandling())
         {
-
         }
 
-
-        internal SonarQubeIssuesProvider(ISonarQubeService sonarQubeService, string sonarQubeProjectKey, ITimerFactory timerFactory,
-            ILogger logger, IThreadHandling threadHandling)
+        internal /* for testing */ SonarQubeIssuesProvider(ISonarQubeService sonarQubeService, string sonarQubeProjectKey,
```

Comment:
Re-ordered the arguments to follow the convention of having the test-only parameters last (i.e. timer factory and thread handling)